src/content/roadmap/maplibre-native/mlt-integration/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: MLT Integration
+heroImage: "./image.png"
+project: maplibre-native
+status: in-progress
+---
+
+Add support for reading vector-tile sources in the [MapLibre Tile Format](https://github.com/maplibre/maplibre-tile-spec) (MLT).
```

Comment:
more fluff

```suggestion
Add support for reading vector-tile sources in the [MapLibre Tile Format](https://github.com/maplibre/maplibre-tile-spec) (MLT).
MLT is a next-generation tile format and a successor to MVT.
Our early benchmarks show 2-6x smaller tiles and some decoding benefits.
```

Reply:
Mentioning benchmarks without a link, reproduction or specifics is not useful IMO. I think we should keep this information in the repo where we can keep it up to date.